apps/client/src/widgets/report/component/graph/graph.tsx

```diff
@@ -0,0 +1,30 @@
+import * as styles from './graph.css';
+
+interface GraphProps {
+  value: 'below' | 'average' | 'above';
+  detailItem?: string;
+  average: number;
+  current: number;
+}
+const Graph = ({ value, average, current, detailItem }: GraphProps) => {
+  return (
+    <div className={styles.container}>
+      {detailItem && <p className={styles.detailItemText}>{detailItem}</p>}
+      <div className={styles.graphExplainContainer}>
+        <div className={styles.textContainer({ value })}>
+          <p className={styles.guaranteeAmountText}>보장 금액</p>
+          <p className={styles.currentAmount}>{current}만원</p>
+        </div>
+        <div className={styles.graphBar}>
+          <div className={styles.graphProgressBar({ value })} />
+        </div>
+        <div className={styles.averageContainer}>
+          <div className={styles.dot} />
```

Comment:
```suggestion
      <span className={styles.dot} aria-hidden="true" />
```

의미를 가지지않는 dot 같은 경우 div 대신 span 태그 + aria-hidden  = true 로 스크린리더에 불필요한 정보를 숨기는 것도 좋아보입니다

Reply:
깊게 생각 안해봤네요.. 너무 좋은 것 같아요. 반영하겠습니다 !